### DISCUSSION TITLE
Please keep HTMLLogo functionality

### DISCUSSION THREAD

**Discussion by ComputerRecon** [EvotecIT/PSWriteHTML#178] [General]
Not sure this is necessary, but after updating PSWriteHTML, saw a notice about New-HTMLLogo being deprecated with a message to let you know if we're using it.  While certainly more customization options would be great, I'm very hopeful that this functionality will remain until a replacement is added.  I do use this function in all of my reports.  I point all my reports to an internet facing logo so it can be visible no matter where accessed.  I generate a multitude of reports for dozens of heathcare sites, so this functionality is perfect in making my reports look cleaner and more professional.  Thanks, and appreciate all the hard work.

**Comment by PrzemyslawKlys** [comment]
Sure. It won't be removed until a new solution is in place. Most likely will be implemented with new navbar functionality so it will take a while to do so. It should already be possible to make a replacement in the current state but it's not as good as I want it to be.

**Reply by Magneet** [reply]
yes please at least keep logo's available in some way!

**Comment by tmeltdowner** [comment]
Does New-HTMLLogo require a jpg or png?  mine doesn't seem to display when using it.

**Reply by PrzemyslawKlys** [reply]
Both should work I guess.

**Comment by raoue** [comment]
I'm responding to the deprecation notice upon using New-HTMLLogo, similar to above. I like having the capability for adding the logo, would be nice for additional features, i.e. centering, sizing without having to do it on the graphic itself, adding additional text with font choices. Anyways, I hope you keep it or something similar in future.
Great work on this module by the way, excellent capabilities. Thanks for your hard work.

**Comment by kieranwalsh** [comment]
Is there any update on the logo functionality? I'd certainly like to keep using it. Cheers.

**Reply by PrzemyslawKlys** [reply]
The new logo functionality is sort of working I believe, but seeing as people use the old approach we may leave it as is, maybe with small updates here and there.

**Reply by kieranwalsh** [reply]
Oh cool, I didn't know a new way - what's that?
cheers for the quick response.

**Reply by PrzemyslawKlys** [reply]
New-HTMLImage and in the nav bars there are a couple of ways. 

```powershell
New-HTML -TitleText 'This is a test' -FilePath "$PSScriptRoot\Example29-01.html" {
    New-HTMLHeader {
        New-HTMLSection -Invisible {
            New-HTMLPanel -Invisible {
                New-HTMLImage -Source 'https://evotec.pl/wp-content/uploads/2015/05/Logo-evotec-012.png' -UrlLink 'https://evotec.pl/' -AlternativeText 'My other text' -Class 'otehr' -Width '50%'
            }
            New-HTMLPanel -Invisible {
                New-HTMLImage -Source 'https://evotec.pl/wp-content/uploads/2015/05/Logo-evotec-012.png' -UrlLink 'https://evotec.pl/' -AlternativeText 'My other text' -Width '20%'
            } -AlignContentText right
        }
    }
    New-HTMLTab -TabName 'Dashboard' {
        New-HTMLTable -DataTable (Get-Process | Select-Object -First 2)
    }
} -ShowHTML -Online
```

![image](https://user-images.githubusercontent.com/15063294/208297859-13760370-29fe-4109-9e8a-282a6d35a680.png)

It's of course not as "easy" but gives you a lot more customization options

```powershell
New-HTML -TitleText 'This is a test' -FilePath "$PSScriptRoot\Example29-01.html" {
    New-HTMLHeader {
        New-HTMLSection -Invisible {
            New-HTMLPanel -Invisible {
                New-HTMLImage -Source 'https://evotec.pl/wp-content/uploads/2015/05/Logo-evotec-012.png' -UrlLink 'https://evotec.pl/' -AlternativeText 'My other text' -Class 'otehr' -Width '50%'
            }
            New-HTMLPanel -Invisible {
                New-HTMLImage -Source 'https://evotec.pl/wp-content/uploads/2015/05/Logo-evotec-012.png' -UrlLink 'https://evotec.pl/' -AlternativeText 'My other text' -Width '20%'
            } -AlignContentText right
        }
    }
    New-HTMLTab -TabName 'Dashboard' {
        New-HTMLTable -DataTable (Get-Process | Select-Object -First 2)
    }
    New-HTMLFooter {
        New-HTMLSection -Invisible {
            New-HTMLPanel -Invisible {
                New-HTMLImage -Source 'https://evotec.pl/wp-content/uploads/2015/05/Logo-evotec-012.png' -UrlLink 'https://evotec.pl/' -AlternativeText 'My other text' -Class 'otehr' -Width '50%'
            }
            New-HTMLPanel -Invisible {
                New-HTMLImage -Source 'https://evotec.pl/wp-content/uploads/2015/05/Logo-evotec-012.png' -UrlLink 'https://evotec.pl/' -AlternativeText 'My other text' -Width '20%'
            } -AlignContentText right
        }
    }
} -ShowHTML -Online
```

![image](https://user-images.githubusercontent.com/15063294/208297897-4aa01aac-a9b6-49b8-a81e-bf6015d730c2.png)

**Reply by kieranwalsh** [reply]
That's perfect, thank you!
